verifier-guest.sh

```diff
@@ -156,6 +161,7 @@ cd ~/geonode
 paver sync
 paver start -b 0.0.0.0:8000
 
+
```

Comment:
Please, remove also this white line.

Reply:
Fixed here: https://github.com/gem/oq-platform2/commit/271a1a8e1c3146c4d6d743e60fb768d436149245